### PR TITLE
fix json parsing

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/internal/ElizaController.scala [hunk 1]
```diff
@@ -40,40 +40,40 @@ class ElizaController @Inject() (
   }
 
   def sendPushNotification() = Action.async { request =>
+    val req = request.body.asJson.get.asInstanceOf[JsObject]
```
Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/internal/ElizaController.scala:43]: `.as[JsObject]`

File: kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/internal/ElizaController.scala [hunk 2]
```diff
@@ -40,40 +40,40 @@ class ElizaController @Inject() (
...
-      val message = (req \ "message").asInstanceOf[String]
-      val pushNotificationExperiment = (req \ "pushNotificationExperiment").asInstanceOf[PushNotificationExperiment]
-      val pushNotificationCategory = (req \ "pushNotificationCategory").asInstanceOf[PushNotificationCategory]
       val deviceCount = messagingCommander.sendPushNotification(userId, message, pushNotificationCategory, pushNotificationExperiment)
       Ok(JsNumber(deviceCount))
     }
   }
 
   def sendToUserNoBroadcast() = Action.async { request =>
+    val req = request.body.asJson.get.asInstanceOf[JsObject]
```
Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/internal/ElizaController.scala:55]: same

**Comment by eishay** [comment]
thanks

**Comment by eishay** [comment]
wow, that's very old code :-)

**Pull request merged**